PWGCF/Femto3D/TableProducer/singleTrackSelector.cxx

```diff
@@ -100,7 +102,7 @@ struct singleTrackSelector {
   std::vector<int> particlesToKeep;
   std::vector<int> particlesToReject;
 
-  void init(InitContext&)
+  void init(InitContext& context)
```

Comment:
Please, don't include `context` as you are not using it
There is a campaign to convert the compiler warning to errors to help to improve the code error detection
Unused variable is one of the warnings that are being removed before converting it to an error